gateway/api/openapi/types.go

```diff
@@ -1068,6 +1068,9 @@ type GuardRailRuleRequest struct {
 		}
 	*/
 	Output map[string]any `json:"output"`
+
+	// List of connection IDs or names that this guardrail applies to
```

Comment:
These are not names, are only connection ids, rename this properly and generate the docs again please?